src/Template/Template.js

```diff
@@ -32,6 +34,28 @@ if (typeof HTMLTemplateElement === 'undefined') {
       while (child = template.firstChild) {
         template.content.appendChild(child);
       }
+      // add innerHTML to template
+      Object.defineProperty(template, 'innerHTML', {
+        get: function() {
+          var o = '';
+          for (var e = this.content.firstChild; e; e = e.nextSibling) {
+            if (e) {
+              o += e.outerHTML || e.textContent;
```

Comment:
fwiw, I think the HTML spec has escaping for text nodes:
https://html.spec.whatwg.org/multipage/syntax.html#serialising-html-fragments
> If current node is a Text node:
> If the parent of current node is a style, script, xmp, iframe, noembed, noframes, or plaintext element, or if the parent of current node is a noscript element and scripting is enabled for the node, then append the value of current node's data IDL attribute literally.
> Otherwise, append the value of current node's data IDL attribute, escaped as described below.

In this case parent is template, so my guess is it should escape. Escape algorithm is:
> Escaping a string (for the purposes of the algorithm above) consists of running the following steps:
> 1. Replace any occurrence of the "&" character by the string `"&amp;"`.
> 2. Replace any occurrences of the U+00A0 NO-BREAK SPACE character by the string `"&nbsp;"`.
> 3. If the algorithm was invoked in the attribute mode, replace any occurrences of the """ character by the string `"&quot;"`.
> 4. If the algorithm was not invoked in the attribute mode, replace any occurrences of the "<" character by the string `"&lt;"`, and any occurrences of the ">" character by the string `"&gt;"`.

Here's an example:
```js
var t = document.createElement('template');
t.innerHTML = 'foo < bar > baz'
t.innerHTML
// prints: "foo &lt; bar &gt; baz"
t.firstChild.textContent
// prints: "foo < bar > baz"
```

And here's some code I wrote or ported from html5lib. (apologies for Dart instead of JS!)
https://github.com/dart-lang/html/blob/317a653cb7ff51c18ef3f632336e1ef72641cc3c/lib/dom_parsing.dart#L192